### PR TITLE
:fist: ¡Viva la Independencia!

### PR DESCRIPTION
**Pull request by lewtun**
Adds the Catalonia Independence Corpus for stance-detection of Tweets.

Ready for review!

### CODE REVIEW

File: datasets/catalonia_independence/catalonia_independence.py [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+        datasets.BuilderConfig(
+            name="spanish", version=VERSION, description="This part of my dataset covers a second domain"
+        ),
+    ]
+
+    DEFAULT_CONFIG_NAME = "catalan"
+
+    def _info(self):
+        features = datasets.Features(
+            {"id_str": datasets.Value("string"), "TWEET": datasets.Value("string"), "LABEL": datasets.Value("string")}
```
Comment (lhoestq) [inline datasets/catalonia_independence/catalonia_independence.py:79]: can you use the ClassLabel Feature type for the LABEL field please ?
For example
```
"LABEL": datasets.ClassLabel(names=["AGAINST", "FAVOR", "NEUTRAL"])
```

you'll need to regenerate the dataset_infos.json file after this change

File: datasets/catalonia_independence/catalonia_independence.py [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+    VERSION = datasets.Version("1.1.0")
+
+    BUILDER_CONFIGS = [
+        datasets.BuilderConfig(
+            name="catalan", version=VERSION, description="This part of my dataset covers a first domain"
+        ),
+        datasets.BuilderConfig(
+            name="spanish", version=VERSION, description="This part of my dataset covers a second domain"
+        ),
+    ]
```
Comment (lhoestq) [inline datasets/catalonia_independence/catalonia_independence.py:73]: can you add descriptions to those configs ?

**Review by lhoestq — COMMENTED** [review]
really cool !

I left a few comments

File: datasets/catalonia_independence/catalonia_independence.py (hunk 1 above)
Comment (lewtun) [reply datasets/catalonia_independence/catalonia_independence.py:79]: thanks for the tip! will add it :)

File: datasets/catalonia_independence/catalonia_independence.py (hunk 2 above)
Comment (lewtun) [reply datasets/catalonia_independence/catalonia_independence.py:73]: good catch!

**Comment by lewtun** [comment]
I've added the changes / fixes - ready for a second pass :)

**Review by lhoestq — APPROVED** [review]
Awesome thank you !

**Pull request merged**